src/Uno.Templates/content/unoapp/MyExtensionsApp.1/Styles/ColorPaletteOverride.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
@carldebilly should we indicate how the file was generate somehow? In a comment maybe it's supported?

Reply:
We could put a note on this in the ReadMe that's included in the project. I wouldn't do it in the actual json file

Reply:
This specific one has not been generated but manually created. We still have [plently](https://platform.uno/docs/articles/external/uno.themes/doc/material-dsp.html) of [documentation about](https://platform.uno/docs/articles/external/figma-docs/learn/designers/dsp-import.html) how to use the [dsp tooling](https://www.nuget.org/packages/Uno.Dsp.Tasks/1.3.2#readme-body-tab).

Reply:
@carldebilly hopefully there are some PRs to update those docs because it's still showing for the old DSP format which is likely to cause confusion now.

Reply:
@dansiegel some documentation needs to be released on the website as well to display the latest changes.
Here is the epic issue for the JSON switch: https://github.com/unoplatform/private/issues/442
If you notice that something is still missing please add it to the list